app/views/calendar/when_do_the_clocks_change.html.erb

```diff
@@ -23,7 +23,7 @@
             } %>
           <% end %>
 
-          <div class="app-c-calendar app-c-calendar--clocks">
+          <div class="app-c-calendar app-c-calendar--clocks govuk-!-margin-bottom-8">
```

Comment:
Since we have a stylesheet for the calendar component, it might be better to include the `margin-bottom` spacing there instead.

Reply:
Good spot, this one is a bit confusing and agree that it should be improved.

Unfortunately this template does not currently used the `calendar` component, even though it appears to be wrapped in the `app-c-` calendar classes. Instead, it simply renders the table component from the gem - https://github.com/alphagov/frontend/pull/4601/files/7dd99bcf9360ff829838d02c1a1b5daedfbf4436#diff-ca24d74025e64b9104a021624c1536aab92612000405bcec4751f2d582133cfdR27, which at present not support margin_bottom.

I can try replacing this with the calendar component, or create a new card for this to be worked on separately.